File: 3169-count-days-without-meetings/3169-count-days-without-meetings.py
class Solution:
    def countDays(self, days: int, meetings: List[List[int]]) -> int:
        answer = 0 
        start = 1

        meetings.sort(key = lambda x : (x[0]))
        
        for s, e in meetings:
            if start < s:
                answer += s - start
            start = max(start, e + 1)

        start-= 1        
        return answer + (days-start)
         

File: 3169-count-days-without-meetings/3169-count-days-without-meetings.js
/**
 * @param {number} days
 * @param {number[][]} meetings
 * @return {number}
 */
var countDays = function(days, meetings) {
    let answer = 0 
    let start = 1
    meetings.sort((a, b)=> a[0]-b[0])
    for (let [s, e] of meetings){
        console.log(start, s, e)
        if (start < s){
            answer += s - start
        }
        start = Math.max(start, e + 1)
    }

    return answer + (days-start+1)
};